bindings/python/htcondor2/_security_context.py

```diff
@@ -0,0 +1,16 @@
+
+
+class SecurityContext():
+
```

Comment:
Given that his appears in the public API, there should be a docstring here.  If you standardize on "PreferredToken", I think it would then suffice rename the `token` parameter of `__init__()`; no explicit documentation is necessary.

You may prefer to put the introductory material (if any) in the docstring for `__init__()`.